### DISCUSSION TITLE
New guest mode exploit

### DISCUSSION THREAD

**Discussion by Exploiter-org** [3kh0/ext-remover#1381] [Exploits JOIN TITANIUM NETWORK TO SHARE PLEASE]
Step 1 powerwash
Step 2 go to the right bottom where it shows ur time click that and press ur internet connect to it and press the setting icon and put custom name servers and add 150.136.163.0 on the first box only. ur gonna wanna wait about 25-30 seconds until a small box in the internet pops up.
Step 3 press get started,then press next. then you'll get an option to browse as guest.
You can press that and you'll be as a guest and nothing will be blocked no policies, no extensions.I don't think the ppl that made the dns exploit knew that their dns could work like this. I think the only reason why it isn't patched it's bc u haven't enrolled yet and when u exit out of guest u can just wait till the enterprise enrollment screen and put ur school email. But what I do is just access guest mode and add my google school account.
Btw if u put that dns u can also  press network sign in and put liberation dashboard and press google and nothings blocked the only thing its a small screen so not very useful compared to the guest mode exploit I found.

**Comment by akabutnicer** [comment]
before everyone goes trying this, @Exploiter-org  what version were you on upon finding this (if you actually did)

**Reply by Exploiter-org** [reply]
130/131

**Comment by Exploiter-org** [comment]
It's not something that can get patched by version

**Reply by Luigi-Master** [reply]
And this should be on a non managed network right?

**Reply by Exploiter-org** [reply]
It works on any network even school if it has the option to do custom name servers

**Comment by K41RO** [comment]
Apparently it just doesn’t connect to the internet for me, and it says it can’t reach google

**Reply by Exploiter-org** [reply]
What version are u in

**Comment by ghost** [comment]
Does not work

**Comment by ghost** [comment]
I don't even get a message or pop up all I get is nothing just connected but no internet

**Comment by ghost** [comment]
Im on version 131.0.6778.96

**Reply by Exploiter-org** [reply]
It only works on 130 or less I just realize it got patched on 131

**Comment by ghost** [comment]
BRUH

**Comment by ghost** [comment]
By the way I need more exploits

**Reply by Exploiter-org** [reply]
Just downgrade, do u have a usb? And personal computer

**Reply by ghost** [reply]
Oh hell nah

**Reply by Exploiter-org** [reply]
?

**Reply by ghost** [reply]
Illegals in my yard ahh music plays hell nah Im not even going to downgrade

**Reply by Exploiter-org** [reply]
Bro what

**Reply by ghost** [reply]
U have more exploits?????

**Reply by Exploiter-org** [reply]
Bro downgrade dawg

**Reply by ghost** [reply]
Nah I'ma cure my down syndrome nah

**Reply by ghost** [reply]
If u gonna post things like this write the version it works on atleast

**Reply by Exploiter-org** [reply]
Like I said I just realized

**Reply by Exploiter-org** [reply]
Bro do u not have the requirements to downgrade or what

**Reply by ghost** [reply]
I have a 40 gb memory chip usb

**Reply by Exploiter-org** [reply]
yes

**Reply by Exploiter-org** [reply]
What's ur discord I'll dm u to help

**Reply by ghost** [reply]
Uhhh I have down syndrome what's discord UHhhhhh

**Reply by Exploiter-org** [reply]
Bro

**Reply by Exploiter-org** [reply]
I'm not gonna help u if ur gonna act stupid

**Comment by ghost** [comment]
will that work?

**Reply by siliconworkshop** [reply]
it will, use chromebook recovery tool
